setup.js

```diff
@@ -0,0 +1,6 @@
+import 'dotenv/config';
+import { User } from './src/models/user.js';
+import { Token } from "./src/models/token.js";
+import { client } from './src/utils/db.js';
+
+client.sync({ alter: true });
```

Comment:
Using `client.sync({ alter: true })` will automatically alter tables to match the model definitions. This is useful for development but can be risky in production as it might lead to data loss. Ensure this is intended for your use case.